examples/text-generation/textgen_pipeline/test_deepspeed.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+import unittest
+
+
+class TestTextGenPipelineDeepSpeed(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        """Overrides setUpClass from unittest to create artifacts for testing"""
+        self.base_command = ["python", "../../gaudi_spawn.py", "--use_deepspeed", "--world_size"]
```

Comment:
`"../.."` will depend on where the test is launched from. I would prefer to have an absolute path here. You can for example use pathlib as follows:
```py
Path(__file__).parent.parent.resolve()
```

Reply:
Will work on this next.